episodes/14-parallel-case-study.md

```diff
@@ -94,23 +92,19 @@ while (c<niter && curdif>=mindif) do {
       for j in coli..colf do {
       ...
 }
-~~~
-{: .source}
+```
 
-As you can see, to divide a data set (the array `temp` in this case) between
-concurrent tasks, could be cumbersome. Chapel provides high-level abstractions
-for data parallelism that take care of all the data distribution for us. We
-will study data parallelism in the following lessons, but for now, let's
-compare the benchmark solution with our `coforall` parallelization to see how
-the performance improved.
+As you can see, to divide a data set (the array `temp` in this case) between concurrent tasks, could be
+cumbersome. Chapel provides high-level abstractions for data parallelism that take care of all the data
+distribution for us. We will study data parallelism in the following lessons, but for now, let's compare the
+benchmark solution with our `coforall` parallelization to see how the performance improved.
 
-~~~
+```bash
 > > chpl --fast parallel_solution_1.chpl -o parallel1
```

Comment:
Don't include prompt characters in bash/shell blocks. If you must, be consistent: `>>` vs. `> >` vs. `$`.

Reply:
Resolved through a commit.